dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/Converter.mo

```diff
@@ -20,7 +20,7 @@ model Converter "Converter model for grid forming and grid following application
 IdcSourcePu     IdcPu |          |iConvPu                           iPccPu
 -------->-------->----|          |-->-----(RFilter,LFilter)---------->--(RTransformer,LTransformer)---(terminal)
               |       |          |                                |
-UdcSourcePu (Cdc)     |  DC/AC   |  uConvPu         uFilterPu (CFilter)                      uPccPu
+UdcSourcePu (Cdc)    |  DC/AC   |  uConvPu         uFilterPu (CFilter)                     uPccPu
```

Comment:
Are you sure of this one ?

Reply:
It is a mistake, the change has been reversed.